src/ray/gcs/gcs_server/gcs_node_manager.h

```diff
@@ -82,7 +82,7 @@ class GcsNodeManager : public rpc::NodeInfoHandler {
                         rpc::CheckAliveReply *reply,
                         rpc::SendReplyCallback send_reply_callback) override;
 
-  void OnNodeFailure(const NodeID &node_id);
+  void OnNodeFailure(const NodeID &node_id, const StatusCallback &callback);
```

Comment:
Could you add docstring for this method and its parameters. 

Also `callback` as variable name is not that meaningful, maybe `node_table_updated_callback`?